examples/plot_visualize_bad_epochs.py

```diff
@@ -87,15 +88,30 @@
 from functools import partial  # noqa
 
 this_epoch = epochs['famous']
-
 exclude = []  # XXX
 picks = mne.pick_types(epochs.info, meg=False, eeg=True, stim=False,
                        eog=False, exclude=exclude)
 
 thresh_func = partial(compute_thresholds, random_state=42, n_jobs=1)
 
+###############################################################################
+# Note that :class:`autoreject.LocalAutoRejectCV` by design supports multiple
+# channels. If no picks are passed separate solutions will be computed for each
+# channel type and internally combines. This then readily supports cleaning
+# unseen epochs from the different channel types used during fit.
+# Here we only use a subset of channels to save time.
+
+###############################################################################
+# Also note that once the parameters are learned, any data can be repaired
+# that contains channels that were used during fit. This also means that time
+# may be saved by fitting :class:`autoreject.LocalAutoRejectCV` on a
```

Comment:
Is this true anymore? I understood that running `transform` on a subsample wasn't working for some reason?